deployer/src/runtime_manager.rs

```diff
@@ -115,19 +131,56 @@ impl RuntimeManager {
         .await
         .context("failed to start shuttle runtime")?;
 
+        let process = Arc::new(Mutex::new(process));
+        let process_cloned = Arc::clone(&process);
         self.runtimes
             .lock()
             .unwrap()
             .insert(id, (process, runtime_client.clone()));
 
+        let stdout = process_cloned
+            .lock()
+            .await
+            .stdout
+            .take()
+            .context("child process did not have a handle to stdout")?;
+
+        // Ensure the child process is spawned in the runtime so it can
+        // make progress on its own while we await for any output.
+        tokio::spawn(async move {
+            process_cloned
+                .lock()
+                .await
+                .wait()
+                .await
+                .expect("child process encountered an error");
+        });
+
+        let mut reader = BufReader::new(stdout).lines();
+        let mut logger_client = self.logger_client.clone();
+        tokio::spawn(async move {
+            while let Some(line) = reader.next_line().await.unwrap() {
+                // TODO: `store_logs` accepts a Vec but logs are sent one by one. Is this feasible?
+                let logs = vec![LogItem {
+                    deployment_id: id.to_string(),
+                    log_line: Some(LogLine {
+                        service_name: service_name.to_string(),
+                        tx_timestamp: Some(Timestamp::from(SystemTime::UNIX_EPOCH)),
```

Comment:
We'll need to use here something like `Utc::now()`.

Reply:
f647dab028b08aa76898aec9bb136ea6116003cb

Reply:
Would be great though to add something like this for nanos: https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.timestamp_subsec_nanos.

Reply:
f2d42cbda13a34558f238b8a6f97c769b51af2d0